reactivex/operators/__init__.py

```diff
@@ -3362,6 +3362,45 @@ def switch_latest() -> Callable[
     return switch_latest_()
 
 
+def switch_map(
```

Comment:
```suggestion
def switch_map_indexed(
```

Reply:
Sure, it's more consistent with RxPy itself. I'll set up both then I guess; a `switch_map` and a `switch_map_indexed`

Reply:
I'm not familiar with the `switch_starmap` operator proposed initially; if needed please let me know, I can look it up and set it up together with tests